docs/dev-guide/integration/rest-api/repo/cud.rst

```diff
@@ -155,6 +155,13 @@ dictate how the importer will function when it synchronizes content. The possibl
 configuration values are contingent on the type of importer being added; each
 importer type will support a different set of values relevant to how it functions.
 
+.. note::
+        There is an optional importer configuration parameter that is usable
+        on all importer plugin types called ``force_full``. If this parameter is set,
+        every sync of the repo using this importer will be done even if the
```

Comment:
what about : "If this parameter is set, a full re-sync will be forced and all steps performed"